File: 7kyu/most_common.rb
def most_frequent_item_count(collection)
    #Hold value of the most counted int
    most_counter = 0
    #iterate over the collection to get first value
    for letter in collection do
        counter = 0
        #compare iterators to count
        for letter_compare in collection do
            if letter == letter_compare
                counter = counter + 1
            end
            #transfer counter to most counted
        if counter > most_counter
            most_counter = counter

        end
        end
    end
    return most_counter
end

File: 7kyu/most_common.py
def most_frequent_item_count(collection):
    highest = 0
    for o_count in collection:
        i_highest = 0
        for i_count in collection:
            if i_count == o_count:
                i_highest += 1
        if i_highest > highest:
            highest = i_highest
    return highest
